### PR TITLE
Add diagnostic for changing OneDrive quota

### PR DESCRIPTION
**Pull request by bpeterse03**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/change-user-storage.md [hunk 1]
```diff
@@ -120,5 +120,15 @@ When you need cloud storage for individual users beyond the initial 5 TB, additi
 
 > [!NOTE]
 > To change the storage space for multiple users, use PowerShell to [Display a list of OneDrive accounts by using PowerShell](list-onedrive-urls.md) and use [Set-SPOSite](/powershell/module/sharepoint-online/set-sposite?preserve-view=true&view=sharepoint-ps&preserve-view=true) to make the change.
-> 
+
+## Diagnose issues with changing the OneDrive storage
+
+> If you are attempting to change the OneDrive storage for a user and are not able to do so or the storage amount reverts to the original value, a Microsoft 365 administrator can run tests to determine what may be preventing the storage from changing.
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:126]:
```suggestion
If you are attempting to change the OneDrive storage for a user and are not able to do so or the storage amount reverts to the original value, a Microsoft 365 administrator can run tests to determine what may be preventing the storage from changing. 
```

File: OneDrive/change-user-storage.md [hunk 2]
```diff
@@ -120,5 +120,15 @@ When you need cloud storage for individual users beyond the initial 5 TB, additi
 
 > [!NOTE]
 > To change the storage space for multiple users, use PowerShell to [Display a list of OneDrive accounts by using PowerShell](list-onedrive-urls.md) and use [Set-SPOSite](/powershell/module/sharepoint-online/set-sposite?preserve-view=true&view=sharepoint-ps&preserve-view=true) to make the change.
-> 
+
+## Diagnose issues with changing the OneDrive storage
+
+> If you are attempting to change the OneDrive storage for a user and are not able to do so or the storage amount reverts to the original value, a Microsoft 365 administrator can run tests to determine what may be preventing the storage from changing. 
+
+>[!NOTE] If you're an administrator, and you're having trouble changing the OneDrive storage for a user or the storage amount reverts to the original value, select **Run Test** below, which will populate the OneDrive Storage Quota diagnostic in the Microsoft 365 Admin Center. These tests will help determine what may be preventing the storage from changing and recommend steps for resolution.
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:128]:
```suggestion
>[!NOTE] If you're an administrator and you're having trouble changing the OneDrive storage for a user or the storage amount reverts to the original value, select **Run Test** below, which will populate the OneDrive storage quota diagnostic in the Microsoft 365 admin center. This test will help determine what may be preventing the storage from changing and recommend steps for a resolution. 
```

File: OneDrive/change-user-storage.md [hunk 3]
```diff
@@ -120,5 +120,15 @@ When you need cloud storage for individual users beyond the initial 5 TB, additi
...
+
+## Diagnose issues with changing the OneDrive storage
+
+> If you are attempting to change the OneDrive storage for a user and are not able to do so or the storage amount reverts to the original value, a Microsoft 365 administrator can run tests to determine what may be preventing the storage from changing. 
+
+>[!NOTE] If you're an administrator, and you're having trouble changing the OneDrive storage for a user or the storage amount reverts to the original value, select **Run Test** below, which will populate the OneDrive Storage Quota diagnostic in the Microsoft 365 Admin Center. These tests will help determine what may be preventing the storage from changing and recommend steps for resolution. 
+>> [!div class="nextstepaction"]
+>> [Run Tests: Check OneDrive Storage Quota](https://aka.ms/PillarOneDriveQuota)
+
+>[!NOTE] This feature is not available for Microsoft 365 Government, Microsoft 365 operated by 21Vianet, or Microsoft 365 Germany.
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:132]:
```suggestion
> [!NOTE]
> This feature is not available for Microsoft 365 Government, Microsoft 365 operated by 21Vianet, or Microsoft 365 Germany. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bpeterse03, kindly commit the suggested changes if you agree with them. Many thanks.

**Pull request merged**